lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -153,23 +153,23 @@
 // -------------------
 .forum-nav-refine-bar {
   @include clearfix();
-  @include font-size(11);
-  border-bottom: 1px solid $gray-l3;
+  font-size: 11px;
```

Comment:
We should use one of our defined size variables, or use rem/em. Also 11px is tiny. 16px should be our minimum. If we absolutely need smaller, 14px.